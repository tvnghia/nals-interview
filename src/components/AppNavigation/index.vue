<template>
  <ul class="nav">
    <router-link tag="li" class="nav__item" to="/" exact> Home </router-link>
    <router-link tag="li" class="nav__item" to="/blogs"> Blogs </router-link>
  </ul>
</template>

<script>
export default {
  name: 'AppNavigation'
}
</script>
