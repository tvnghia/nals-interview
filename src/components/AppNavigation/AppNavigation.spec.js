import { shallowMount } from '@vue/test-utils'
import wrapperOps from '@/config-test/wrapper'
import AppNavigation from '@/components/AppNavigation'

describe('AppNavigation.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(AppNavigation, wrapperOps)
  })

  afterEach(() => { wrapper.vm.$destroy() })

  describe('when AppNavigation was mounted', () => {
    it('should AppNavigation render correctly', () => {
      expect(wrapper.exists()).toBeTruthy()
      expect(wrapper).toMatchSnapshot()
    })
  })
})
