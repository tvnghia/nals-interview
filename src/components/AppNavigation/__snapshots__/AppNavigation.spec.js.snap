// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavigation.vue when AppNavigation was mounted should AppNavigation render correctly 1`] = `
<ul class="nav">
  <router-link-stub to="/" tag="li" exact="true" ariacurrentvalue="page" event="click" class="nav__item">Home</router-link-stub>
  <router-link-stub to="/blogs" tag="li" ariacurrentvalue="page" event="click" class="nav__item">Blogs</router-link-stub>
</ul>
`;
